src/wifi_hal_nl80211.c

```diff
@@ -4837,30 +4846,42 @@ static int wiphy_dump_handler(struct nl_msg *msg, void *arg)
     struct nlattr *tb[NL80211_ATTR_MAX + 1];
     struct genlmsghdr *gnlh;
     //unsigned int *cmd;
-#if !defined(VNTXER5_PORT)
     unsigned int j;
-#endif
     unsigned int phy_index = 0;
+#ifndef FEATURE_SINGLE_PHY
     int rdk_radio_index;
+#else //FEATURE_SINGLE_PHY
+    int rdk_radio_indices[MAX_NUM_RADIOS];
+    int num_radios_mapped = MAX_NUM_RADIOS;
+    int ret = 0;
+#endif //FEATURE_SINGLE_PHY
+
 #if defined(VNTXER5_PORT)
     int existing_radio_found = 0;
 #endif
+#ifndef FEATURE_SINGLE_PHY
     if (g_wifi_hal.num_radios > MAX_NUM_RADIOS) {
+#else //FEATURE_SINGLE_PHY
+    if (g_wifi_hal.num_radios >= MAX_NUM_RADIOS) {
+#endif //FEATURE_SINGLE_PHY
+        wifi_hal_dbg_print("%s:%d: Returning num radios:%d exceeds MAX:%d\n",
+            __func__, __LINE__, g_wifi_hal.num_radios, MAX_NUM_RADIOS);
         return NL_SKIP;
     }
 
-    gnlh = nlmsg_data(nlmsg_hdr(msg));
-
-    nla_parse(tb, NL80211_ATTR_MAX, genlmsg_attrdata(gnlh, 0), genlmsg_attrlen(gnlh, 0), NULL);
-
 #if !defined(VNTXER5_PORT)
     for (j = 0; j < g_wifi_hal.num_radios; j++)
     {
         if (strcmp(g_wifi_hal.radio_info[j].name, nla_get_string(tb[NL80211_ATTR_WIPHY_NAME])) == 0) {
+            wifi_hal_dbg_print("%s:%d: Returning phy:%s already configured earlier\n",
+                __func__, __LINE__, g_wifi_hal.radio_info[j].name);
             return NL_SKIP;
         }
     }
 #endif
+    gnlh = nlmsg_data(nlmsg_hdr(msg));
```

Comment:
This 3 lines 4882 -4884 should go above the #if !defined(VNTXER5_PORT) as per the coverity error https://jenkins.ccp.xcal.tv/jenkins/job/TECHNICOLORXB8-Yocto-Pipeline/23313/artifact/build-images/coverity_dir/html/1/65_wifi_hal_nl80211.c.html#error

Reply:
Done